.github/workflows/test.yml

```diff
@@ -0,0 +1,183 @@
+name: "[CI] anaconda-cli-base"
```

Comment:
Do you think we should rename this file? I will need to select a workflow to publish to PyPI by name.

We could use a generic `ci.yml`, or we could look into splitting this into `test.yml` and `publish.yml`. I do still want to make sure the build jobs pass before a PR is merged. We can defer that refactor too.

What do you think?

Reply:
done